src/backend/src/form-responses/form-responses.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { BadRequestException, Body, Controller, Post } from '@nestjs/common';
+import { Question } from 'src/entities/question.entity';
+import { User } from 'src/entities/user.entity';
+import { FormResponse } from 'src/entities/formResponse.entity';
+
+@Controller('form-responses')
+export class FormResponsesController {
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('questionId') questionId: number,
+    @Body('answer') answer: string,
+  ): Promise<FormResponse> {
+    const formResponse = new FormResponse();
+    const user = await User.findOne(userId);
+    const question = await Question.findOne(questionId);
+    if (!user || !question) {
+      throw new BadRequestException('Invalid userId or questionId');
+    }
+    formResponse.user = user;
+    formResponse.question = question;
+    await formResponse.save();
+    return formResponse;
```

Comment:
I'm just thinking this now, but it might be convenient to have a second post method like @Post('allResponses') which could post multiple questions at once. That way we won't have to call post over and over when they finish the questions page

Reply:
Wait each person will only fill out the written response once?

Reply:
I was under the impression that there was a second set of non long-form questions, but i could have imagined that.

Reply:
So there are demographic questions as well as reflection questions. Let's add a POST postAll() for the demographic and reflection questions @lizzij

Reply:
Awesome cool, glad i did not imagine that. That's why we added the questionType field to formResponse (i think)

Reply:
Yeah, that makes sense. Do you think we should be using DTO (Data Transfer Object)?

Reply:
Maybe Aislin can speak to this more, but I think that would make sense? Would we have to define a FormResponseDTO and also LoFormResponseDTO tho?

Reply:
Yes, I think we can just have exposed entities for now and add the DTOs if necessary later.

Reply:
wait i think we should add the DTOs now, and not leave it to tech debt

Reply:
Sounds good. I'll get started and try to refactor user with DTO as well.